### PR TITLE
Tweak precalc FN4 checkit

### LINKED ISSUE #281: FN4 CheckIt
- For reflections, change the language to "reflected over the x-axis" or "reflected over the y-axis" instead of "flipped". 
- Could the two graphs be shown less zoomed in? And on graphs with the same  range of x's and y's so it's more clear the changes that happened? Without paying attention to the scales closely, the horizontal and vertical shifts are not very apparent.
- And, this may be a general thing more than just this particular one, but the scaling of the axes to fit sort of loses perspective of where things are on the coordinate plane. I think seeing a more standard -10,10 (or -8 to 8) for the x's and y's would be better.

### PR DESCRIPTION
**Pull request by siwelwerd**
Closes #281 

@AbbyANoble does this look better?

image
image

### CODE REVIEW

**Comment by AbbyANoble** [comment]
Yes! But that's a reflection across the x-axis, not y.

File: source/precalculus/exercises/outcomes/FN/FN4/template.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
             <p>
                 To produce <m>{{gname}}(x)</m>, the graph of <m>{{fname}}(x)</m>is shifted {{hshift}} by <m>{{b}}</m>,
                 <!--{{#flip}}-->
-                flipped vertically,
+                reflected across the <m>y</m>-axis,
```
Comment (siwelwerd) [inline source/precalculus/exercises/outcomes/FN/FN4/template.xml:28]:
```suggestion
                reflected across the <m>x</m>-axis,
```

**Comment by siwelwerd** [comment]
Github really needs to implement the 🤦🏻‍♂️ emoji as a reaction.

**Comment by AbbyANoble** [comment]
It was the y-values that changed though. I'm sure that's what you were thinking of. 😁

**Comment by AbbyANoble** [comment]
Hey @StevenClontz! Can you review this and the other CheckIt pull requests so they are updated (and I can use them next week!)?

**Comment by StevenClontz** [comment]
Actually I need to make it so *you* can review these! I'm going ahead and merging these so you can look at them in the morning yourself; report back in a new Issue if there's any concerns.

**Review by StevenClontz — APPROVED** [review]

**Pull request merged**